### ISSUE TITLE
Add Dark mode support

### ISSUE THREAD

**Issue by umair313** [jmrivas86/django-json-widget#81]
* django-json-widget version:
* Django version:
* Python version:
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by StevenMapes** [comment]
Duplicate to #59

**Comment by jmrivas86** [comment]
Fixed in new 2.0.0 version
